bundles/org.openhab.binding.guntamatic/README.md

```diff
@@ -291,8 +291,8 @@ String               Biostar_ProgramHc5              "Program HC5"
 String               Biostar_ProgramHc6              "Program HC6"                        { channel="guntamatic:biostar:mybiostar:status#076-program-hc6" }
 String               Biostar_ProgramHc7              "Program HC7"                        { channel="guntamatic:biostar:mybiostar:status#077-program-hc7" }
 String               Biostar_ProgramHc8              "Program HC8"                        { channel="guntamatic:biostar:mybiostar:status#078-program-hc8" }
-String               Biostar_Interuption0            "Interuption 0"                      { channel="guntamatic:biostar:mybiostar:status#079-interuption-0" }
-String               Biostar_Interuption1            "Interuption 1"                      { channel="guntamatic:biostar:mybiostar:status#080-interuption-1" }
+String               Biostar_Interruption0           "Interruption 0"                     { channel="guntamatic:biostar:mybiostar:status#079-interruption-0" }
```

Comment:
see above

Reply:
Similar response, this is the name (user created) of an item and label name. This is in user space, not at all related to the api and I don’t see why we should propagate this spelling mistake in item example. (Obviously not talking about the channel id, I can agree to stick to inherit that mistake)